groups/sig-testing/groups.yaml

```diff
@@ -169,6 +169,32 @@ groups:
   # required for membership in these groups.
   #
 
+  - email-id: k8s-ci-robot@kubernetes.io
```

Comment:
just to confirm, this is https://groups.google.com/a/kubernetes.io/g/k8s-ci-robot/ which is *NOT* public read?

Reply:
`WhoCanPostMessage: "ANYONE_CAN_POST"` is only needed to ensure we can receive e-mails.

Reply:
Yeah, but I don't see the config for "who can read this". 
The live link appears that it is restricted, wanted to confirm.